sysutils/gptfdisk/Portfile

```diff
@@ -0,0 +1,39 @@
+# $Id:  $
+
+PortSystem              1.0
+
+name                    gptfdisk
+version                 1.0.1
+categories              sysutils
+platforms               darwin
+license                 GPL-2+
+maintainers             mail.ru:nano103
```

Comment:
The maintainers field should also list your GitHub username:
```
maintainers {@nano103 mail.ru:nano103}
```

Consider adding `openmaintainer` to [allow others to update](https://guide.macports.org/#project.update-policies.nonmaintainer) the port.